CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [3.20.3] - 2023-04-21
 - Added tests for config file updates, updated logic so that async writes and locking wait do not cause the loss of data added in previous writes.
 - Added more error handling for cloud watch logs so that it gracefully handles errors and does not interrupt users.
+- Skip Healthcheck Test and when server is down shows connection error in bufferHandler catch block.
```

Comment:
@syedgulahmed I think v3.20.3 has been released. Please put this under v3.20.4. Thanks

Reply:
Also update the version in build.gradle file @syedgulahmed

Reply:
Also update the version in build.gradle file @syedgulahmed